moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -2174,6 +2175,59 @@ void RobotState::printStatePositions(std::ostream& out) const
     out << nm[i] << "=" << position_[i] << std::endl;
 }
 
+bool RobotState::printStatePositionsWithJointLimits(const moveit::core::JointModelGroup* jmg, std::ostream& out) const
```

Comment:
What's the motivation behind the return value in this print function?
The function is always returning true. Please change to returning `void`.

Reply:
it used to return false when there were multi-variable joints, but i removed that. fixed per you